Resources/Prototypes/Entities/Mobs/Player/admin_ghost.yml

```diff
@@ -4,6 +4,10 @@
   name: admin observer
   noSpawn: true
   components:
+  - type: Eye
+    visMask:
+      - Ghost
+      - Normal
```

Comment:
uhhh is this necessary?

Reply:
That was necessary back when invisible aghosts and non invisible aghosts could still see eachother. With that being removed, this is no longer needed.